### PR TITLE
Mohammad/ Sign-up-success to TS

### PR DESCRIPTION
**Pull request by mohammad-rework**
Changes:

-   Convert Sign-up-success page to TS

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [x] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
         "jsx": "preserve",
         "esModuleInterop": true,
         "experimentalDecorators": true,
         "emitDecoratorMetadata": true,
         "noEmit": true,
         "skipLibCheck": true,
-        "moduleResolution": "node"
-      }
+        "moduleResolution": "node",
+        "lib": ["dom", "dom.iterable", "ES2021.String"]
```
Comment (yashim-deriv) [inline tsconfig.json:13]: Any reason for adding these types for built-in JS APIs?

Comment (mohammad-rework) [reply tsconfig.json:13]: > Any reason for adding these types for built-in JS APIs?

- Add `"lib": ["dom", "dom.iterable"]` to support `URLSearchParams(location.search)` in src/signup-success/index.tsx line 53.
- Add `"lib": ["ES2021.String"]` to support `.replaceAll` method in src/signup-success/index.tsx line 57.

**Review by yashim-deriv — APPROVED** [review]
LGTM

**Comment by sean-binary** [comment]
There's a conflict @mohammad-rework 🙏

**Review by sean-binary — APPROVED** [review]

**Pull request merged**